vendor/golang.org/x/sys/unix/zsysnum_linux_amd64.go

```diff
@@ -353,4 +353,6 @@ const (
 	SYS_FSPICK                 = 433
 	SYS_PIDFD_OPEN             = 434
 	SYS_CLONE3                 = 435
+	SYS_OPENAT2                = 437
+	SYS_PIDFD_GETFD            = 438
```

Comment:
SYS_OPENAT2 looks related to https://patchwork.kernel.org/patch/11167585/

- should we have these vendor-bumps in a separate commit? (are they needed for 19.03.x?)
- do we need to add `openat2` to our seccomp profile? https://github.com/moby/moby/blob/0f41a77c6993ade605a639fb25994cfe5e1b3fe8/profiles/seccomp/seccomp_default.go#L224

Reply:
Thanks, let me take a look.

Reply:
I've separated these commits. 

And I will open a PR for add `openat2` to  our seccomp profile.

Reply:
Opened #41353